### DISCUSSION TITLE
Problema Sonoff Dongle E e Home assistant

### DISCUSSION THREAD

**Discussion by Gradom** [indomus/forum#2349] [Hardware & Firmware]
Ciao a tutti, sto avendo problemi con la Sonoff in questione. Praticamente ho un pc dedicato con Proxmox e una vm con Home assistant. Funzionava tutto bene sino a quando la sonoff è sparita da HA. unica soluzione era riavviare il pc, non bastava riavviare la sola istanza. La cosa era frequente. Cosi ho deciso di usare il mio portatile personale, con Fedora, e ho installato KVM e virtualizzato HA, ma non ho usato backup precedenti,  ma completamente una nuova istanza. Fatto ciò la Sonoff funzionava, ma dopo 1 giorno e una notte, la mattina al mio risveglio, la Sonoff era sprita di nuovo dalla lista hardware di HA, ma su fedora con comendo "lsusb" risultava connessa, e anche in KVM c'è. COsa posso fare? dove sbaglio?

**Comment by KC-inDomus** [comment]
> Ciao a tutti, sto avendo problemi con la Sonoff in questione. Praticamente ho un pc dedicato con Proxmox e una vm con Home assistant. Funzionava tutto bene sino a quando la sonoff è sparita da HA. unica soluzione era riavviare il pc, non bastava riavviare la sola istanza. La cosa era frequente. Cosi ho deciso di usare il mio portatile personale, con Fedora, e ho installato KVM e virtualizzato HA, ma non ho usato backup precedenti, ma completamente una nuova istanza. Fatto ciò la Sonoff funzionava, ma dopo 1 giorno e una notte, la mattina al mio risveglio, la Sonoff era sprita di nuovo dalla lista hardware di HA, ma su fedora con comendo "lsusb" risultava connessa, e anche in KVM c'è. COsa posso fare? dove sbaglio?

la versione firmware è l'ultima?

**Reply by Gradom** [reply]
Sisi ultima ma il problema persiste..ho brasato anche proxmox e messo HA come os direttamente..mi è successo che dopo ore la pennetta sparisce dall'hardware e quando è presente il plugin z2m non si collega alla sonoff e quindi non parte

**Reply by KC-inDomus** [reply]
ma non è semplicemente che ti vanno in idle le porte usb per power saving

**Reply by Gradom** [reply]
Potrebbe essere..vanno controllate da bios?

**Reply by KC-inDomus** [reply]
in realtà potrebbe essere gestito anche da sistema operativo

**Comment by Gradom** [comment]
Mmmh ho provato a girare sulle impostazioni di home assistant e non ho trovato nulla e anche dalla cli non sono riuscito a settare nulla
